### ISSUE TITLE
Parsing packing version numbers from container tag

### ISSUE THREAD

**Issue by kchaung-lilly** [seqeralabs/wave#716]
Hello, in our group, we have users building many similar Wave containers. However, the package versions are no longer in the container names, and it is difficult to see what's inside without entering and inspecting. 

Is there any way to parse the package version names from the tag hash?
For example, in `community.wave.seqera.io/library/bbmap_pigz_seqkit_csvtk:fa8c3b82bba97878`, can fa8c3b82bba97878 be used to determine what are the versions of bbmap, pigz, seqkit, and csvtk?

Thank you!

**Comment by pditommaso** [comment]
Yes, you can use this uri `https://wave.seqera.io/view/builds/{id}` for example https://wave.seqera.io/view/builds/fa8c3b82bba97878

**Comment by kchaung-lilly** [comment]
Thanks!

I specifically was referring to the use case of when we have already pushed a copy of the container into our ECR, so the container name may look like <AWS_ENV>/wave/containers/bbmap_pigz_seqkit_csvtk:fa8c3b82bba97878. At this point, we may no longer be able to retrieve the build id.

**Comment by pditommaso** [comment]
wave strips the version numbers from the tag when there's more than one tools because it could be quire unreadable. 

An option could be include in the image name, but not sure how much it will improve the problem
